fix: add parallel structure (#175)
open-sauced/docs: docs/community/100-days-of-oss.md

Reviewer: Consider omitting this part

The beauty of this challenge is that you're not required to code. The main purpose is to grow in your understanding of open source software (OSS), contribute in ways that are meaningful to you, and further develop the skills and knowledge you're interested in pursuing. It's all about personal growth and making a positive impact on the OSS community. Because this challenge is focused on growth, you can participate in any way that helps you achieve your goals-including taking days off when you need it.

There are numerous ways to participate in the #100DaysOfOSS challenge, including:
### Content creation
Love creating content? Here's how you can use this skill for the challenge:
  - Give a talk or presentation on OSS.
  - Participate in or even start a Twitter Space.
  - Write a blog post or create a video.
### Project management
Want to improve your project management skills? Need to get better at communicating or giving feedback to others? Here's how you can use this skill for the challenge:
  - Review pull requests.
  - Triage issues.

> To keep track of your progress, post on social media, your blog, or any other platform you prefer with what progress you made, the day of the challenge indicated by 'D'and include the hashtag #100DaysOfOSS. For example, if you're on day one, you could say, "Today, I reviewed the documentation for the [OpenSauced/Insights](https://github.com/open-sauced/insights) repository. D1 #100daysOfOSS." Then, on day two, you would continue with D2, and so on.

Reviewer: [proofreading/suggestions] Consider adding a heading so participants can see what is being discussed. 
```suggestion
### Tracking your progress
To keep track of your progress, post on social media, your blog, or any other platform you prefer with what progress you made, the day of the challenge indicated by 'D'and include the hashtag #100DaysOfOSS. For example, if you're on day one, you could say, "Today, I reviewed the documentation for the [OpenSauced/Insights](https://github.com/open-sauced/insights) repository. D1 #100daysOfOSS." Then, on day two, you would continue with D2, and so on.
```

Author: > [proofreading/suggestions] Consider adding a heading so participants can see what is being discussed.

Done.

> - write issues to identify bugs or suggest new features;
> - triage existing issues to help with prioritization;
> - submit pull requests to contribute code changes;

Reviewer: [proofreading] Omit this part
```suggestion

```

Author: Done

- write issues to identify bugs or suggest new features;
- triage existing issues to help with prioritization;
- submit pull requests to contribute code changes;

> - engage in the community by sharing your insights and knowledge; 

Reviewer: Omit this part too @RitaDee

> - support contributors working on OSS projects, providing guidance and support.

Reviewer: [nitpick] You can omit this too
```suggestion

```

> - create or participate in discussions related to OSS topics;
> - write a blog post or create content that supports an OSS project; 

Reviewer: [nitpick]Since you've listed these two points, I recommend omitting this part for organizational reasons. 
```suggestion 
```

Author: Okay

> - update or write documentation to improve clarity and usability

Reviewer: [nitpick] Omit this part too
```suggestion
```

> - Content creation: Love creating content? Here's how you can use this skill for the challenge:
>   - Give a talk or presentation on OSS.
>   - Participate in or even start a Twitter Space.
>   - Write a blog post or create a video.
> 
> - Project management: Want to improve your project management skills? Need to get better at communicating or giving feedback to others? Here's how you can use this skill for the challenge:
>   - Review pull requests.
>   - Triage issues.
>   - Respond to questions.
> - support contributors working on OSS projects, providing guidance and support.

Reviewer: ```suggestion
### Content creation
Love creating content? Here's how you can use this skill for the challenge:
  - Give a talk or presentation on OSS.
  - Participate in or even start a Twitter Space.
  - Write a blog post or create a video.

### Project management
Want to improve your project management skills? Need to get better at communicating or giving feedback to others? Here's how you can use this skill for the challenge:
  - Review pull requests.
  - Triage issues.
  - Respond to questions.
- support contributors working on OSS projects, providing guidance and support.
```

Author: Noted with thanks, @CBID2

> To keep track of your progress, post on social media, your blog, or any other platform you prefer with what progress you made, the day of the challenge indicated by 'D'and include the hashtag #100DaysOfOSS. For example, if you're on day one, you could say, "Today, I reviewed the documentation for the [OpenSauced/Insights](https://github.com/open-sauced/insights) repository. D1 #100daysOfOSS." Then, on day two, you would continue with D2, and so on.

Reviewer: [nitpick/proofreading] This part is off to a great start! :) Good idea to provide an example of how participants can post their progress. I think it can be improved by mentioning the word "contributions" and separating the example to the bottom as shown below: 
```suggestion
To keep track of your progress, post about your contributions on social media, your blog, or any other platform you prefer. Also, add the day of the challenge indicated by the initial 'D' and include the hashtag `#100DaysOfOSS`. For example, if you're on day one, you could do the following:
"Today, I reviewed the documentation for the [OpenSauced/Insights](https://github.com/open-sauced/insights) repository. D1 #100daysOfOSS." Then, you would do the same for the second day, and so on.
```
